clients/stellar-relay-lib/src/connection/handshake.rs

```diff
@@ -14,7 +17,8 @@ pub enum HandshakeState {
 }
 
 pub fn create_auth_message() -> StellarMessage {
-	let auth = Auth { flags: 100 };
+	// 200 for flowControlInBytes enabled.
+	let auth = Auth { flags: AUTH_FLAG };
```

Comment:
Should we always enable it? I can see [here](https://github.com/stellarbeat/js-stellar-node-connector/blob/9120f24b75867700ba0ece76369166959205c6b9/src/connection/connection.ts#L634C9-L634C22),  [here](https://github.com/stellarbeat/js-stellar-node-connector/blob/9120f24b75867700ba0ece76369166959205c6b9/src/connection/handshake-message-creator.ts#L12) that it should be enabled if the overlay version of the node is `>= 28` so we could simplify it to the statement here, but maybe it's better if we instead use a condition as they do in the JS code. If that's too much refactoring, we can also adjust the comment to something like
```
// AUTH_FLAG is set to `200` to enable flow control in bytes when connecting to nodes with overlay version >= 28. 
// We assume that the overlay version is always >= 28
```

Reply:
Sure, I'll try since it doesn't look like too much effort.